IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the stuck rate of tickets in the turnstile.
```

Comment:
```suggestion
Indicate the frequency of tickets being stuck or jammed in appliance..
```

Reply:
Updated, but changed "frequency" to "ratio". https://github.com/bSI-InfraRoom/IFC-Specification/pull/160/commits/8b12068ba20d88a43f933b941b15b54fce5e5c4d